plugins/modules/ndo_tenant_igmp_interface_policy.py

```diff
@@ -0,0 +1,508 @@
+    type: str
+  description:
+    description:
+    - The description of the IGMP Interface Policy.
+    - Providing an empty string will remove the O(description="") from the IGMP Interface Policy.
+    type: str
+  version3_asm:
+    description:
+    - Enable or disable IGMP version 3 ASM.
+    - Defaults to C(disabled) when unset during creation.
+    type: str
+    choices: [ enabled, disabled ]
+    aliases: [ allow_version3_asm ]
+  fast_leave:
+    description:
+    - Enable or disable fast leave.
+    - Defaults to C(disabled) when unset during creation.
+    type: str
+    choices: [ enabled, disabled ]
+  report_link_local_groups:
+    description:
+    - Enable or disable reporting link-local groups.
+    - Defaults to C(disabled) when unset during creation.
+    type: str
+    choices: [ enabled, disabled ]
+  igmp_version:
+    description:
+    - The IGMP version of the IGMP Interface Policy.
+    - Defaults to C(v2) when unset during creation.
+    type: str
+    choices: [ v2, v3 ]
+  group_timeout:
+    description:
+    - The group timeout value in seconds.
+    - Defaults to C(260) when unset during creation.
+    - The valid range is from C(3) to C(65535).
+    type: int
+  query_interval:
+    description:
+    - The query interval value in seconds.
+    - Defaults to C(125) when unset during creation.
+    - The valid range is from C(1) to C(18000).
+    type: int
+  query_response_interval:
+    description:
+    - The query response interval value in seconds.
+    - Defaults to C(10) when unset during creation.
+    - The valid range is from C(1) to C(25).
+    type: int
+  last_member_count:
+    description:
+    - The last member query count value.
+    - Defaults to C(2) when unset during creation.
+    - The valid range is from C(1) to C(5).
+    type: int
+  last_member_response_time:
+    description:
+    - The last member query response time value in seconds.
+    - Defaults to C(1) when unset during creation.
+    - The valid range is from C(1) to C(25).
+    type: int
+  startup_query_count:
+    description:
+    - The startup query count value.
+    - Defaults to C(2) when unset during creation.
+    - The valid range is from C(1) to C(10).
+    type: int
+  startup_query_interval:
+    description:
+    - The startup query interval value in seconds.
+    - Defaults to C(31) when unset during creation.
+    - The valid range is from C(1) to C(18000).
+    type: int
+  querier_timeout:
+    description:
+    - The querier timeout value in seconds.
+    - Defaults to C(255) when unset during creation.
+    - The valid range is from C(1) to C(65535).
+    type: int
+  robustness_variable:
+    description:
+    - The robustness variable value.
+    - Defaults to C(2) when unset during creation.
+    - The valid range is from C(1) to C(7).
+    type: int
+  state_limit_route_map_uuid:
+    description:
+    - The state limit route map name.
```

Comment:
Is the description correct? The option ends with `_uuid`?

Reply:
changed to UUID